config.w32

```diff
@@ -140,6 +151,14 @@ if (PHP_MONGODB != "no") {
     WARNING("mongodb libsasl support not enabled, libs not found");
   }
 
+  if (typeof COMPILER_NAME === 'string') {
+    mongoc_opts.MONGOC_CC = COMPILER_NAME;
+  } else if (typeof VC_VERSIONS[VCVERS] === 'string') {
+    mongoc_opts.MONGOC_CC = VC_VERSIONS[VCVERS];
```

Comment:
What does this to the output? I'd rather have it being consistent, than "proper". So if VC_VERSIONS works for everything, then we should use that (unless it does nothing to the result of course).